server/models/Collective.js

```diff
@@ -164,6 +165,19 @@ function defineModel() {
         },
       },
 
+      /** Private, legal name. Used for expense receipts, taxes, etc. */
+      legalName: {
+        type: Sequelize.STRING,
```

Comment:
Maybe `DataTypes.STRING` just to be consistent as we use `DataTypes` all throughout the model files, although I think this will work as well. 🤔